scss/custom/_photo-gallery.scss

```diff
@@ -12,7 +12,8 @@
     opacity: 1;
 
     &:focus-visible > .material-symbols-rounded {
-      box-shadow: $btn-focus-box-shadow;
+      @include border-radius(2px);
+      box-shadow: 0 0 0 .25rem rgba(184, 43, 65, .5);
```

Comment:
Is this a custom color? I see it in other places in this PR: would it be good to make a custom variable for it?

Reply:
It's 50% opacity brand red, I believe